roles/icinga_agent/README.md

```diff
@@ -8,9 +8,10 @@ This role installs and configures the icinga agent.
 | icinga_agent_package | yes | icinga2 | installs icinga2 package
 | icinga_agent_registration | yes | false | used to register your installed icinga agent against your icinga master
 | icinga_agent_enable_features | no | | config files for extra features you can use along with icinga. If you add for example api.conf to the variable the feature will be active in your icinga instance
-| icinga_agent_ca_host | no | | your master intance (f.e: master0-example.de)
+| icinga_agent_ca_host | no | | your master instance (f.e: master0-example.de)
 | icinga_agent_ca_host_icinga_port | no | 5665 | Icinga agent port
 | icinga_agent_salt | no | | used to hash password
+| icinga_agent_hostname | no | "{{ ansible_hostname }}" | define hostname (use `ansible_fqdn` for icinga satellites)
```

Comment:
> (use `ansible_fqdn` for icinga satellites)

I have troubles understanding that comment.

Reply:
Icinga Conventions states that all nodes should be configured using its FQDN. [Conventions](https://icinga.com/docs/icinga-2/latest/doc/06-distributed-monitoring/#conventions)

To avoid breaking existing installations, default should stay on `ansible_hostname`.

Reply:
Woah. This comment packs quite some information. From what I understand we should change the default to `ansible_fqdn` but this would be a breaking change. First we need to be able to configure this to move on with our project, so we should take this intermediate step.

Can you rephrase the coment a bit:
```suggestion
| icinga_agent_hostname | no | "{{ ansible_hostname }}" | define hostname (icinga satellites require this to be set to `ansible_fqdn` or some other usable FQDN)
```

After we merge this, we should create a follow-up issue to make the breaking change in the default setting.